tests/src/test_files/test_files.resource

```diff
@@ -5,4 +5,5 @@ Documentation       Test files data
 ${test_files_directory_path}  ${CURDIR}${/}
 &{file_4mb}  name=file_4MB.exe  path=${test_files_directory_path}file_4MB.exe  archive_size_bytes=${4722512}  archive_name=smol11  files_in_archive=2
 &{10k_files}  name=10k_files.exe  path=${test_files_directory_path}10k_files.exe  archive_size_bytes=${42537871}  archive_name=10000 koni wielkości kaczki  files_in_archive=10001
-&{multi_part_4mb}  name=multi_part_4MB  path=${test_files_directory_path}multi_part_4MB  archive_size_bytes=${4722512}  archive_name=smol2  files_in_archive=2  parts=4      
+&{multi_part_4mb}  name=multi_part_4MB  path=${test_files_directory_path}multi_part_4MB  archive_size_bytes=${4722512}  archive_name=smol2  files_in_archive=2  parts=4
+&{file_4mb_corrupt}  name=file_4MB_corrupt.exe  path=${test_files_directory_path}file_4MB_corrupt.exe  archive_size_bytes=${4722512}  archive_name=smol11  files_in_archive=2
```

Comment:
Please make sure that ${test_files_directory_path} variable will be updated to ${input_test_files_path} according to recent changes in 'tests' branch. Just adding this comment to make sure this will be updated since you added a new line and it might not be catch as a conflict before merge.

Reply:
Done